accounting.js

```diff
@@ -0,0 +1,49 @@
+const log4js = require('log4js');
+const logger = log4js.getLogger('<filename>');
+log4js.configure({
+    appenders: {
+        file: { type: 'fileSync', filename: 'logs/debug.log' }
+    },
+    categories: {
+        default: { appenders: ['file'], level: 'debug'}
+    }
+});
+const moment = require('moment');
+
+function doAccounts(dataFile) {
+    let namesList = [];
+    for (let i = 0; i < dataFile.length; i++) {

```

Comment:
Most (all?) of these `for` loops are just iterating over arrays; given that it'd be a little cleaner to use `forEach`, e.g.:

    dataFile.forEach(dataLine => {
      namesList.push(dataLine.From);
      namesList.push(dataLine.To);
    })